### PR TITLE
[feat-5577]: filter disabling

### PR DESCRIPTION
**Pull request by vdegraaf**
Ticket: [SIG-5577](https://gemeente-amsterdam.atlassian.net/browse/SIG-5577)

note: this is a subtask, therefore merging with feature branch

[SIG-5577]: https://gemeente-amsterdam.atlassian.net/browse/SIG-5577?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/Notification/index.tsx [hunk 1]
```diff
@@ -107,18 +107,10 @@ const Notification: FunctionComponent<NotificationProps> = ({
...
-
-      /* istanbul ignore else */
-      if (typeof onClose === 'function') {
-        onClose()
-      }
-    }, ONCLOSE_TIMEOUT)
-
-    slideUpTimeoutRef.current = slideUpTimeoutId
+    /* istanbul ignore else */
+    if (typeof onClose === 'function') {
```
Comment (OscarBakker) [inline src/components/Notification/index.tsx:111]: isn't onClose typed? if it is, these type of runtime validations seem a bit to much to me

File: src/signals/incident-management/containers/IncidentOverviewPage/index.js [hunk 2]
```diff
@@ -66,12 +69,45 @@ export const IncidentOverviewPageContainerComponent = ({
...
   const showsMap = location.pathname.split('/').pop() === MAP_URL
+  const hasActiveOrdering =
+    ordering && ordering !== '-created_at' && ordering !== 'created_at'
+
+  const totalPages = useMemo(() => Math.ceil(count / FILTER_PAGE_SIZE), [count])
+
+  const canRenderList = results && results.length > 0 && totalPages > 0
+
+  /* istanbul ignore next */
+  const hasActiveFilters = activeFilter.options
```
Comment (OscarBakker) [inline src/signals/incident-management/containers/IncidentOverviewPage/index.js:90]: more ninja: const hasActiveFilters = activeFilter.options
  ? Object.values(activeFilter.options).some((arr) => arr.length > 0)
  : false;

**Review by OscarBakker — APPROVED** [review]
Approved, just look at a few points

**Comment by vdegraaf** [comment]
@OscarBakker all the comments are on code I didn't touch. They've just been replaced in the same file. So I'd rather not refactor them here.

**Pull request merged**